dbt_copilot_helper/commands/copilot.py

```diff
@@ -262,7 +274,7 @@ def _generate_env_addons(
     for addon in addon_template_map[addon_type].get("env", []):
         template = templates.get_template(addon["template"])
 
-        contents = template.render({"service": environment_addon_config, "addons": addons})
+        contents = template.render({"addon_config": environment_addon_config, "addons": addons})
```

Comment:
This has been bugging me for a while, been meaning to do a separate pull request for it, but I couldn't leave it any more.